server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -1633,6 +1632,7 @@ private void tryToDeleteLastCache(PartialPath deviceId, PartialPath originalPath
           if (lastPair != null && startTime <= lastPair.getTimestamp()
               && lastPair.getTimestamp() <= endTime) {
             ((MeasurementMNode) measurementNode).resetCache();
+            logger.info("[tryToDeleteLastCache] Last cache for path: " + measurementNode.getFullPath() + " is set to null");
```

Comment:
we could use placeholders for log-print...

Reply:
Fixed